website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,554 @@
+The `iam:PassedToService` restricts the permission to make these requests to only the EC2 service (`ec2.amazonaws.com`).
+
+```
+            {
+              "Sid": "AllowPassingInstanceRole",
+              "Effect": "Allow",
+              "Resource": "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/KarpenterNodeRole-${ClusterName}",
+              "Action": "iam:PassRole",
+              "Condition": {
+                "StringEquals": {
+                  "iam:PassedToService": "ec2.amazonaws.com"
+                }
+              }
+            },
+```
+
+### AllowAPIServerEndpointDiscovery
+The Karpenter controller needs to be able to find the Kubernetes cluster's API endpoint in order to communicate with the API server.
+The AllowAPIServerEndpointDiscovery Sid allows the Karpenter controller to get that information (`eks:DescribeCluster`) for the cluster (`cluster/${ClusterName}`).
+```
+            {
+              "Sid": "AllowAPIServerEndpointDiscovery",
+              "Effect": "Allow",
+              "Resource": "arn:${AWS::Partition}:eks:${AWS::Region}:${AWS::AccountId}:cluster/${ClusterName}",
+              "Action": "eks:DescribeCluster"
+            }
+          ]
+        }
+```
+
+
+
+# Interruption Handling 
+Settings in this section allow the Karpenter controller to interact with interruption queues.
+So, for example, if Spot instances are being reclaimed or a node crashes, seeing messages from these queues allows Karpenter to be proactive in moving workloads or adding new nodes.
+Defining the `KarpenterInterruptionQueuePolicy` allows Karpenter to see and respond to the following:
+
+* AWS health events
+* Spot interruptions
+* Spot rebalance recommendations
+* Instance state changes
+
+The resources defined in this section include:
+
+* KarpenterInterruptionQueue
+* KarpenterInterruptionQueuePolicy
+* ScheduledChangeRule
+* SpotInterruptionRule
+* RebalanceRule
+* InstanceStateChangeRule
+
+## KarpenterInterruptionQueue
+The [AWS::SQS::Queue](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sqs-queue.html) resource is used to create an Amazon SQS standard queue.
+Properties of that resource set the `QueueName` to the name of your cluster, the time for which SQS retains each message (`MessageRetentionPeriod`) to 300 seconds, and enabling serverside-side encryption using SQS owned encryption keys (`SqsManagedSseEnabled`) to `true`.
+See [SetQueueAttributes](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SetQueueAttributes.html) for descriptions of some of these attributes.
+
+```
+  KarpenterInterruptionQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      QueueName: !Sub "${ClusterName}"
+      MessageRetentionPeriod: 300
+      SqsManagedSseEnabled: true
+```
+
+## KarpenterInterruptionQueuePolicy
+The Karpenter interruption queue policy is created and applied to allow the Karpenter controller to see messages for selected services.
```

Comment:
This queue policy actually just defines what services can push to the queue. It has nothing to do with Karpenter's ability to read from it (that's defined in the KarpenterControllerPolicy). I'd say something like "The Karpenter interruption queue policy is created to allow AWS services that we want to receive instance notifications from to push notification messages to the queue."

Reply:
Done.